backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConnectorAsUsageImpl.java

```diff
@@ -320,7 +334,11 @@ public void setIsDirected(boolean newIsDirected) {
      */
     @Override
     public EList<Feature> getRelatedFeature() {
-        List<Feature> data = new ArrayList<>();
+        List<Feature> data = this.getConnectorEnd().stream()
+                .map(Feature::getOwnedReferenceSubsetting)
+                .filter(rs -> rs != null)
```

Comment:
Sorry I didn't see it before, please prefer `.filter(Objects::nonNull)` instead of `.filter(rs -> rs != null)`

Reply:
Yes, you are right, I've changed the ConnectorImpl code as well (I did align those two classes).